config/locales/subscriptions_management.yml

```diff
@@ -1,10 +1,28 @@
 en:
   subscriptions_management:
+    heading: Manage your subscriptions
     index:
       subscription:
         immediately: "You chose to get updates as soon as they happen."
         daily: "You chose to get daily updates."
         weekly: "You chose to get weekly updates."
     update_address:
+      heading: Change your email address
+      current_email: "Your current email address is %{address}"
+      description: |
+        "If you want to transfer your subscription to %{address},
```

Comment:
I think we need to drop the quotes here, as otherwise they'll be part of the printed text.

Reply:
Done